packages/legacy/src/common/constants.ts

```diff
@@ -107,9 +107,9 @@ export enum StepType {
 export const dnsLabelNameSchema = yup
   .string()
   .max(63)
-  .matches(/^[a-z0-9][a-z0-9-]*[a-z0-9]$/, {
+  .matches(/^(?=.{1,253}$)[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/, {
     message: ({ label }) =>
-      `${label} can only contain lowercase alphanumeric characters and dashes (-), and must start and end with an alphanumeric character`,
+      `${label} can only contain lowercase alphanumeric characters, dashes and dots, and must start and end with an alphanumeric character, see k8s documentation for more details.`,
```

Comment:
I suggest using a similar validation text message for all entities names.

The text message I used for the Provider name validation is:
`Error: Name is required and must be a unique within a namespace and valid Kubernetes name (i.e., must contain no more than 253 characters, consists of lower case alphanumeric characters , '-' or '.' and starts and ends with an alphanumeric character).`

Reply:
sure, but remember to use ${label} and not Name

Reply:
> together with the Providers name text message

ahh, didn't notice that part, please make it a separate PR